php-frontend/src/main/java/org/sonar/php/parser/TreeFactory.java

```diff
@@ -0,0 +1,1100 @@
+      InternalSyntaxToken unsetToken, InternalSyntaxToken openParenthesisToken,
+      ExpressionTree expression, Optional<List<Tuple<InternalSyntaxToken, ExpressionTree>>> list,
+      InternalSyntaxToken closeParenthesisToken, InternalSyntaxToken eosToken
+  ) {
+    return new UnsetVariableStatementTreeImpl(
+        unsetToken,
+        openParenthesisToken,
+        separatedList(expression, list),
+        closeParenthesisToken,
+        eosToken
+    );
+  }
+
+  public NamespaceStatementTree namespaceStatement(InternalSyntaxToken namespaceToken, NamespaceNameTree namespaceName, InternalSyntaxToken eosToken) {
+    return new NamespaceStatementTreeImpl(
+        namespaceToken,
+        namespaceName,
+        eosToken
+    );
+  }
+
+  public NamespaceStatementTree blockNamespaceStatement(
+      InternalSyntaxToken namespaceToken, Optional<NamespaceNameTree> namespaceName,
+      InternalSyntaxToken openCurlyBrace, Optional<List<StatementTree>> statements, InternalSyntaxToken closeCurlyBrace
+  ) {
+    return new NamespaceStatementTreeImpl(
+        namespaceToken,
+        namespaceName.orNull(),
+        openCurlyBrace,
+        optionalList(statements),
+        closeCurlyBrace
+    );
+  }
+
+  public InlineHTMLTree inlineHTML(InternalSyntaxToken inlineHTMLToken) {
+    return new InlineHTMLTreeImpl(inlineHTMLToken);
+  }
+
+  public DeclareStatementTree shortDeclareStatement(DeclareStatementHead declareStatementHead, InternalSyntaxToken eosToken) {
+    return new DeclareStatementTreeImpl(declareStatementHead, eosToken);
+  }
+
+  public DeclareStatementHead declareStatementHead(
+      InternalSyntaxToken declareToken, InternalSyntaxToken openParenthesisToken,
+      VariableDeclarationTree firstDirective, Optional<List<Tuple<InternalSyntaxToken, VariableDeclarationTree>>> optionalDirectives,
+      InternalSyntaxToken closeParenthesisToken
+  ) {
+    return new DeclareStatementHead(
+        declareToken,
+        openParenthesisToken,
+        separatedList(firstDirective, optionalDirectives),
+        closeParenthesisToken
+    );
+  }
+
+  public DeclareStatementTree declareStatementWithOneStatement(DeclareStatementHead declareStatementHead, StatementTree statement) {
+    return new DeclareStatementTreeImpl(declareStatementHead, statement);
+  }
+
+  public DeclareStatementTree alternativeDeclareStatement(
+      DeclareStatementHead declareStatementHead, InternalSyntaxToken colonToken,
+      Optional<List<StatementTree>> statements,
+      InternalSyntaxToken enddeclareToken, InternalSyntaxToken eosToken
+  ) {
+    return new DeclareStatementTreeImpl(declareStatementHead, colonToken, optionalList(statements), enddeclareToken, eosToken);
+  }
+
+  public StaticStatementTree staticStatement(InternalSyntaxToken staticToken, VariableDeclarationTree variable, Optional<List<Tuple<InternalSyntaxToken, VariableDeclarationTree>>> listOptional, InternalSyntaxToken eosToken) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 225 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00103)